src/app/portlets/content-types/fields/content-type-fields-properties-form/content-type-fields-properties-form.component.ts

```diff
@@ -119,15 +134,25 @@ export class ContentTypeFieldsPropertiesFormComponent extends BaseComponent impl
         this.setAutoCheckValues();
     }
 
+    private isPropertyDisabled(property: string): boolean {
+        return (
+            this.fieldPropertyService.isDisabledInEditMode(property) ||
+            this.fieldPropertyService.isDisabledInFixed(this.formFieldData.fixed, property)
```

Comment:
In order to keep the pattern we were handling here, I think is better to do

```|| this.formFieldData.fixed && this.fieldPropertyService.isDisabledInFixed(property)```

But I want comments from @alfredo-dotcms and @freddyucv on this.

Reply:
it looks more such as business question!